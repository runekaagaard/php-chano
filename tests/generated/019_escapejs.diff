001- paragraph separator:\u2029and line separator:\u2028
001+ paragraph separator:\u000Aand line separator:\u000A