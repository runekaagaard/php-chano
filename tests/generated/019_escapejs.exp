paragraph separator:\u2029and line separator:\u2028