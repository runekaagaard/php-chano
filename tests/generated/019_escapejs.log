
---- EXPECTED OUTPUT
paragraph separator:\u2029and line separator:\u2028
---- ACTUAL OUTPUT
paragraph separator:\u000Aand line separator:\u000A
---- FAILED
