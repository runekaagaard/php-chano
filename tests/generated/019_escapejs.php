<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'paragraph separator:
and line separator:
')));
foreach ($items as $i) echo $i->input->escapejs();
