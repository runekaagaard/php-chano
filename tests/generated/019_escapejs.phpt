--TEST--
A generated testfile for the "escapejs" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'paragraph separator:
and line separator:
')));
foreach ($items as $i) echo $i->input->escapejs();
--EXPECT--
paragraph separator:\u2029and line separator:\u2028