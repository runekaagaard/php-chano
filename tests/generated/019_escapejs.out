paragraph separator:\u000Aand line separator:\u000A